tests/testthat/test_get_receivers.R

```diff
@@ -22,10 +22,12 @@ testthat::test_that("test_output_get_receivers", {
   expect_is(test1, "data.frame")
   expect_is(test2, "data.frame")
   expect_is(test3, "data.frame")
-  expect_gte(nrow(test1), nrow(test2))
+  expect_gt(nrow(test1), nrow(test2))
   expect_gte(nrow(test1), nrow(test3))
```

Comment:
Can we then coherently change this `expect_gte` to `expect_gt` as well? Not such a big difference at the end of course.

Reply:
no, should be checked for each, I would leave it as such for now